pilot/pkg/model/jwks_resolver.go

```diff
@@ -345,7 +345,7 @@ func (r *JwksResolver) getRemoteContentWithRetry(uri string, retry int) ([]byte,
 			return nil, err
 		}
 
-		if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
```

Comment:
In such comparisons I think having numbers may be more clear - this is my personal preference. But let us wait for others feedback as well.

Reply:
> In such comparisons I think having numbers may be more clear - this is my personal preference. But let us wait for others feedback as well.


I agree with you ，let me recovert it